src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -55,7 +55,7 @@ const Confirmation = React.forwardRef<HTMLElement, Props>((
 ) => {
   const drawFocusRef = useDrawFocus();
   return <Overlay
-    ref={mergeRefs(ref, drawFocus ? drawFocusRef : null)}
+    ref={mergeRefs(ref, drawFocusRef)}
```

Comment:
Leaving this ternary in dropped code coverage. I'm not sure how, since I see it gets called with and without `drawFocus={false}` in the code. I've put it back in, and will see if I need to write a new test.
![image](https://github.com/openstax/rex-web/assets/7315241/ab1ffe37-77e4-45e8-badd-73bbd5109cfa)

Reply:
💡  The EditCard mocks the Confirmation dialog, and the Confirmation spec only tests with defautl `drawFocus` of true.